### DISCUSSION TITLE
Why are most new signatories Russian?

### DISCUSSION THREAD

**Discussion by ghost** [rms-support-letter/rms-support-letter.github.io#5310] [General]
Did this petition get posted someplace Russian?

**Comment by nekonee** [comment]
We welcome signers of all nationalities here.

**Comment by grindelwaldus** [comment]
yep, it was posted here - https://habr.com/ru/news/t/548718/

**Comment by nukeop** [comment]
https://www.linux.org.ru/news/opensource/16225630
https://www.linux.org.ru/forum/talks/16225882?cid=16226448

**Comment by nuclearcat** [comment]
It got published on russian opensource news website https://www.opennet.ru/opennews/art.shtml?num=54820

**Comment by RollingHog** [comment]
Well, we do want to use free software too.

**Comment by majestrate** [comment]
believe it or not free software also exists outside of the USA.

**Comment by df3423e** [comment]
because russians are based

**Comment by EnverOsmanov** [comment]
Spreading communism in software world!

**Comment by Areso** [comment]
> Spreading communism in software world!

Yeah, somewhat.
For everyone for the person's needs, from everyone from the person's abilities.
It sounds good, isn't it?

**Comment by majestrate** [comment]
> Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior.

that's kind of racist ngl, just because someone is russian doesn't make them a bot.

**Comment by Areso** [comment]
@BurnyBoi and who should be counted as a bot?
A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?

**Comment by RollingHog** [comment]
> @BurnyBoi and who should be counted as a bot?
> A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?

Hey, just step aside and let this boi BURN!

**Comment by nukeop** [comment]
plonked

**Comment by Koncord** [comment]
@BurnyBoi I'm Russian and I did a lot more effort to the Open Source than you

**Comment by ghost** [comment]
Because we love free software

**Comment by ghost** [comment]
In Russia, we also like free software and respect RMS and all of he's done.

**Comment by mfriley** [comment]
Americans discovering for the billionth time that the world doesn't revolve around them and their politics

**Comment by shpinog** [comment]
> @BurnyBoi and who should be counted as a bot?
> A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?

The presence of some projects, no matter how significant, does not give you the right to decide where to work for a person and to privatize his right to personal opinion.

**Comment by pizdjuk** [comment]
> > @BurnyBoi and who should be counted as a bot?
> > A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?
> 
> The presence of some projects, no matter how significant, does not give you the right to decide where to work for a person and to privatize his right to personal opinion.

Just to be honest: the retoric question of BurnyBoy referring to russians: "Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior."

So, the question of "What did you do for free software?" is pretty in place :)

**Comment by evdmux** [comment]
it was posted in 2ch.hk . its the most popular anonymous image board in Russian.

**Comment by podvornyakva** [comment]
Cause we are wake early, to seek God mercy.

**Comment by ghost** [comment]
Excellent. :sunglasses:

**Comment by Watcher3056** [comment]
Stalin and Lenin respects free software, legate!

**Comment by shpinog** [comment]
> > > @BurnyBoi and who should be counted as a bot?
> > > A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?
> > 
> > 
> > The presence of some projects, no matter how significant, does not give you the right to decide where to work for a person and to privatize his right to personal opinion.
> 
> Just to be honest: the retoric question of BurnyBoy referring to russians: "Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior."
> 
> So, the question of "What did you do for free software?" is pretty in place :)

Users define value too, not just developers.

RMS and the open source software movement are targeting users.

You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?

**Comment by ghost** [comment]
Ahhh didn't mean to close this, wrong button, sorry! Didn't know free software was such a big thing in Russia.

**Comment by Areso** [comment]
> You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?

It is not about limiting anybody or anywhere.
It is about value, as you said.

Let's say, a stranger came into a bar and said 'this pal is a cool marksman'. Okay.
But his words will be valued more if he is by himself is a marksman too, and he is well-known for that fact.
So, basically, everyone could have an opinion on everything.
But not everybody's opinion is worth something.

**Comment by longyap** [comment]
rms blessed the world with freedom

**Comment by pizdjuk** [comment]
> > > > @BurnyBoi and who should be counted as a bot?
> > > > A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?
> > > 
> > > 
> > > The presence of some projects, no matter how significant, does not give you the right to decide where to work for a person and to privatize his right to personal opinion.
> > 
> > 
> > Just to be honest: the retoric question of BurnyBoy referring to russians: "Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior."
> > So, the question of "What did you do for free software?" is pretty in place :)
> 
> Users define value too, not just developers.
> 
> RMS and the open source software movement are targeting users.
> 
> You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?

The limiting of personal opinion is exactly what intents the rms-open-letter xD So, anyone who signed the open antiRMS letter can not ask me "who gives me the right to limit a persons personal opinion?"

**Comment by shpinog** [comment]
> > You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?
> 
> It is not about limiting anybody or anywhere.
> It is about value, as you said.
> 
> Let's say, a stranger came into a bar and said 'this pal is a cool marksman'. Okay.
> But his words will be valued more if he is by himself is a marksman too, and he is well-known for that fact.
> So, basically, everyone could have an opinion on everything.
> But not everybody's opinion is worth something.

That's it. A good shooter also has a right to his opinion, as does a beginner. To whom to listen to this matter of an individual person, he also has the opportunity not to listen. But someone is trying to prove that a good shooter has the right to decide where to work and how to think to other shooters.

**Comment by pizdjuk** [comment]
> Did this petition get posted someplace Russian?

BTW, don't forget, that russian language is one of the most speaked language in the world. Beginning from the territorium of exUSSR (which includes dozens of countries) en ending in Israel and some parts of Near Orient.

**Comment by Watcher3056** [comment]
> 
> 
> > You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?
> 
> It is not about limiting anybody or anywhere.
> It is about value, as you said.
> 
> Let's say, a stranger came into a bar and said 'this pal is a cool marksman'. Okay.
> But his words will be valued more if he is by himself is a marksman too, and he is well-known for that fact.
> So, basically, everyone could have an opinion on everything.
> But not everybody's opinion is worth something.

Lets say Stallman WORLD(not just our) value is not comparable greather to the reason why someones want to push him out. And i think there is enough peoples understanding the REAL reasons why some forces want to remove him from FSF and GNU. And no body care about Baiden son leaks, yeah? What a weird world we live in...

**Comment by artemgur** [comment]
Because the topic went viral on https://habr.com, Russian (mainly) website about tech. Many people came from there. When the general awareness rises, there will be more people from all the world

**Comment by shpinog** [comment]
> > > > > @BurnyBoi and who should be counted as a bot?
> > > > > A person who has contributed nothing, like you, or this bunch of Russians, who had a lot of open source projects behind their names?
> > > > 
> > > > 
> > > > The presence of some projects, no matter how significant, does not give you the right to decide where to work for a person and to privatize his right to personal opinion.
> > > 
> > > 
> > > Just to be honest: the retoric question of BurnyBoy referring to russians: "Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior."
> > > So, the question of "What did you do for free software?" is pretty in place :)
> > 
> > 
> > Users define value too, not just developers.
> > RMS and the open source software movement are targeting users.
> > You still evade the question, what right does anyone have to limit a person's personal opinion? Having a cool repository gives you the right to decide who should think how and where to work?
> 
> The limiting of personal opinion is exactly what intents the rms-open-letter xD So, anyone who signed the open antiRMS letter can not ask me "who gives me the right to limit a persons personal opinion?"

Do you see me on the signed list?

It turns out the level of these "cool developers", at the right time to stick your head in the sand like an ostrich with the words "I won't answer you because I don't like you"?

**Comment by pizdjuk** [comment]
Did you asked me _that_ question? I didnt see it. I answered to comment to BurnyBoi and in those context. So, if you are not in a list of those signants of rms-open-letter, and, I suppose, you want to ask me this question, than I can answer you. Nobody gives me the right to limit _you_ in your opinion.

**Comment by JoeMcNuggets** [comment]
Because we are communists.

**Comment by dexpl** [comment]
> Because we are communists.

Please speak for yourself, OK?

**Comment by JoeMcNuggets** [comment]
@dexpl, no.

**Comment by pizdjuk** [comment]
> > Because we are communists.
> 
> Please speak for yourself, OK?

Who told you that he^W they speaks not for himself^W themselves? ;)

**Comment by KOLANICH** [comment]
@evdmux,

>2ch.hk
>mail.ru group
>anonymous

/0

**Comment by shpinog** [comment]
> Did you asked me _that_ question? I didnt see it. I answered to comment to BurnyBoi and in those context. So, if you are not in a list of those signants of rms-open-letter, and, I suppose, you want to ask me this question, than I can answer you. Nobody gives me the right to limit _you_ in your opinion.

I proceed from the assumption that all people are equal in their basic rights. Therefore, if I have the right to do this, then RMS too.

Today you prohibit the opinion of RMS, indicate where to work. Tomorrow you will tell me how to think and where to work.

The day after tomorrow you will decide who has the right to use open source software.

You undermine the foundations of openness with your inhibitions.

**Comment by pizdjuk** [comment]
@shpinog 

Aaaahhh. I think you counfounds me with somebody else :) I'm on the side of RMS and his free opinion.

**Comment by TimurC1** [comment]
Потому что опубликовали на Хабре, Опеннете, ЛОРе и куче наших любимых ресурсов. И да, мы за свободу программного обеспечения, а также нас не волнуют политические и другие высказывания, за которые Ричарда сейчас хотят отстранить.

**Comment by pizdjuk** [comment]
Ваистену!

**Comment by 13sqrt5** [comment]
Since this turned into general discussion: your predictions on the number of signatures?

**Comment by TimurC1** [comment]
> Since this turned into general discussion: your predictions on the number of signatures?

Вроде форков почти столько же, сколько и у проприетарщиков, а пулл реквестов мало

**Comment by TimurC1** [comment]
> Ahhh didn't mean to close this, wrong button, sorry! Didn't know free software was such a big thing in Russia.

Большинство населения нашей страны даже не знает элементарные правила защиты данных и т.п., попадают на удочки мошенников. Но так как страна большая, то хоть и процент нас, просвещенных в плане СПО, мал, нас много

**Comment by Areso** [comment]
> Since this turned into the general discussion: your predictions on the number of signatures?

by the end of the week - 3 grand top.
A significant amount of devs will not sign this, because they are afraid that the hate wave will target them, after Stallman.
One of the pros of being Russian - they don't give a dim about those SJWs, so they don't need to look around before signing a person they respect.

**Comment by TimurC1** [comment]
> Because we love free software

Ну, у нас любят большинство населения все-таки FREE в значении "БЕСПЛАТНОЕ", а не свободное.

**Comment by AKonia** [comment]
> > Because we love free software
> 
> Ну, у нас любят большинство населения все-таки FREE в значении "БЕСПЛАТНОЕ", а не свободное.

Ну за всю страну уж не наглей говорить, бесплатное любят все, но вот как раз на безвоздмездную, искреннюю помощь и добро для всех - делают у нас много, чаще, это результат другого воспитания: помогаешь другу = помогаешь себе.

**Comment by AKonia** [comment]
> > Ahhh didn't mean to close this, wrong button, sorry! Didn't know free software was such a big thing in Russia.
> 
> Большинство населения нашей страны даже не знает элементарные правила защиты данных и т.п., попадают на удочки мошенников. Но так как страна большая, то хоть и процент нас, просвещенных в плане СПО, мал, нас много

Повторю: не своди к общему свои частности.

**Comment by podvornyakva** [comment]
> by the end of the week - 3 grand top.

3 grans give easily only russiаn community. That is my bet.
UP: Let's say it will be around 6. Double or nothing. ^_^

**Comment by pizdjuk** [comment]
Ребята, предлагаю удалить эту политоту.

**Comment by KOLANICH** [comment]
I propose to translate all the comments in Russian into English. Not everyone in international community knows Russian.

@AKonia, @TimurC1

**Comment by pizdjuk** [comment]
>I propose to translate all the comments in Russian into English. Not everyone in international community knows Russian.

Who wants to read, will translate by his own account. Not everyone knows English too. So what? They translate English into whatever will be....

**Comment by centralhardware** [comment]
here it is better to write a TL;DR  with a context than to make a translation

**Comment by AKonia** [comment]
@KOLANICH not necessary, because commonly it just useless offtopic opinions, at the same time offtopic can be discussed on any language with no harm... because it useless.

**Comment by pizdjuk** [comment]
@AKonia Да есть просто товарищи, которые как-то странно относятся к своему родному языку... Я на всякий случай сходил за дополнительной порцией поп-корна :)

**Comment by KOLANICH** [comment]
@AKonia, I know :) But members of int community don't. Please show responsibility and translate, especially if you allowed yourself to post offtop.

**Comment by tuxarch** [comment]
https://github.com/igoradamenko/awesome-made-by-russians

**Comment by pizdjuk** [comment]
Пускай учат русский, аахахаххахах xD

**Comment by KOLANICH** [comment]
@pizdjuk, GH is an international website, the petition targets the whole world. It is just irresponsible and disrespectful to make a private party here. For offtop you can find other places, such as opennet and lor.

**Comment by 13sqrt5** [comment]
TL;DR of Russian babble:
While in Russia small percent of people are computer-savvy, among the educated people the support for Open Source is very strong.  Since the country is reasonably large, even a small fraction is a lot of people. Then there was a discussion regarding that many Russians understand free software as in "free beer", not as in "free speech".

**Comment by akurichenko** [comment]
Not only Russian.
I'm from Ukraine and I also respect RMS and Free Software

**Comment by pizdjuk** [comment]
> @pizdjuk, GH is an international website, the petition targets the whole world. It is just irresponsible and distespectful to make междусобойчик (untranslateable noun meaning something that is for the members of a group only) here. For offtop you can find other places, such as opennet and lor.

меня там забанили xD Русский, если что, сходит в шестерку (?) языков ООН. Устраивать междусобойчик может быть irresponsible, но уж никак не disrespectful. А вообще -- нормал. Давайте чаду!! Ахххахахаха.

Переводить не буду  (да и не умею, если честно). Незачем засорять взор многоуважаемого иностранца подобной нечистью :)

**Comment by AKonia** [comment]
@KOLANICH language is needed to communicate with other people and i suppose if someone wants to communicate with concrete people off topic - they are free to do that.

**Comment by pizdjuk** [comment]
А вообще -- тред-детектор :)

**Comment by AKonia** [comment]
@13sqrt5 no, some thinks of FOSS as free-beer and the most of Russians as I know thinks about it as some form a sincere act of kindness - when you help others, you help yourself

**Comment by 13sqrt5** [comment]
@AKonia I did not express my opinion on that matter, i merely summed up what was written in Russian.

**Comment by AKonia** [comment]
@13sqrt5 some of them(msgs) was just toxic drops and reactions to it.

**Comment by AKonia** [comment]
@KOLANICH междусобойчик - private party/group. There are no untranslatable thing.

**Comment by KOLANICH** [comment]
The translation is not exact, but I guess it would fit here. Thanks.

**Comment by HAUSEVULT** [comment]
Lefties at it again with their anti-white racism. Let me guess, all signatories **must** be [group who approves of my thinking]?

**Comment by AKonia** [comment]
@HAUSEVULT righties not lefties.

**Comment by shpinog** [comment]
> @pizdjuk, GH is an international website, the petition targets the whole world. It is just irresponsible and disrespectful to make a private party here. For offtop you can find other places, such as opennet and lor.

Откуда вы все берётесь? У вас здесь прям party любителей указывать другим как им жить. Люди и без "особого мнения" разберутся где и для чего "своё" место.

**Comment by ghost** [comment]
> > Ahhh didn't mean to close this, wrong button, sorry! Didn't know free software was such a big thing in Russia.
> 
> Большинство населения нашей страны даже не знает элементарные правила защиты данных и т.п., попадают на удочки мошенников. Но так как страна большая, то хоть и процент нас, просвещенных в плане СПО, мал, нас много

Это же можно сказать про всех людей

**Comment by 13sqrt5** [comment]
@AKonia
@HAUSEVULT
Please refrain from blaming broad "lefties" or "righties", as the signatories of this letter may consider themselves "left" or "right".

**Comment by AKonia** [comment]
@13sqrt5 i just reacted with only remark. At the same time I think that this issue should be frozen from the beginning(or right after first 2-3 comments), wanna or not - it will end as political talk because it's began as offtopic, mixing national with international.

**Comment by TimurC1** [comment]
> > > Because we love free software
> > 
> > 
> > Ну, у нас любят большинство населения все-таки FREE в значении "БЕСПЛАТНОЕ", а не свободное.
> 
> Ну за всю страну уж не наглей говорить, бесплатное любят все, но вот как раз на безвоздмездную, искреннюю помощь и добро для всех - делают у нас много, чаще, это результат другого воспитания: помогаешь другу = помогаешь себе.

Не буду с вами спорить, все таки мы в адекватном обществе. Я уважаю ваше мнение. Но моя точка зрения такова, я это заметил по собственным наблюдениям. Это мое личное мнение 🙂

**Comment by TimurC1** [comment]
> > > Ahhh didn't mean to close this, wrong button, sorry! Didn't know free software was such a big thing in Russia.
> > 
> > 
> > Большинство населения нашей страны даже не знает элементарные правила защиты данных и т.п., попадают на удочки мошенников. Но так как страна большая, то хоть и процент нас, просвещенных в плане СПО, мал, нас много
> 
> Это же можно сказать про всех людей

Можете уточнить?

**Comment by trdm** [comment]
> Well, we do want to use free software too.

А так-же любим его разрабатывать :)

**Comment by 1vanK** [comment]
@BurnyBoi

> Because it's bots inflating the signature count to try and make it seem as relevant as the real letter exposing his bad behavior.

Based on your profile, bot are you.

**Comment by Gobz** [comment]
Never knew free software was this big in Russia! Glad to see you guys on board. Don't take too much to the bigots calling you bots.

**Comment by maxp779** [comment]
Welcome Russian friends 😊

**Comment by pizdjuk** [comment]
> > Well, we do want to use free software too.
> 
> А так-же любим его разрабатывать :)

звучит, кхм, занятно :) Даже не знаю, что повлияло.... Наверное резкая смена языка xD

**Comment by ghost** [comment]
Not a Russian but Vietnamese, glad to see fellow comrades supporting free software.

**Comment by 6r1d** [comment]
It's great to see this issue, really. Many of us respect RMS for projects he created like GCC. I expected that less people from Russia will be interested in supporting RMS and this is really reassuring.

I'm personally here because I saw [this video](https://open.lbry.com/@Lunduke:e/DefenseofStallman:9?r=GAaJ9s3EZNYqp61oaAXUcUzgEMPmhhf1) by Bryan Lunduke, if that matters. Apparently, there is a topic on opennet and I'm reading another one on linux.org.ru now.

**Comment by ghost** [comment]
GCC is part of my motivation too. My entire country's programming scene depends on GCC, and I cannot let a couple hundred angry Twitter imbeciles take the freedom of being able to build software away from us.

**Comment by nuclearcat** [comment]
There is one more reason i realize, why so many russians present here.
It's funny and sad, but Russian-speaking supporters of RMS have more freedom of speech at moment, they are not subject of  treacherous revenge from those who are trying to lynch RMS.

**Comment by Bluesboy** [comment]
Because in Russia we honestly believe in freedom of expression and despise those who against it.

**Comment by michaelkarlcoleman** [comment]
I for one welcome our new Russian bot overlords.

**Comment by DOORAK** [comment]
Loud emptiness laying behind points antiRMS letter, we can see clearly. Because staying far and being in distant edges of media-field gap - we not bounded by social-relation mind refractors causing people to call white balls as "black balls" repeatedly after society group leaders calling it as that for they reasons ("Asch conformity experiments").

**Comment by ghost** [comment]
> I for one welcome our new Russian bot overlords.

All Rusians are bots, arent they? Seems similar to racism

**Comment by thatHguy** [comment]
Us in the West have completely lost the moral high ground of being more "free" since the events of 2020. Russia, China, USA, Europe, etc. We are all together now facing world-encompassing totalitarianism spearheaded by the tech companies. Many of the anti-FSF mob are backed by these corporations. Cancelling FSF is being done to diminish the digital civil liberties movement.

**Comment by kotovalexarian** [comment]
> Spreading communism in software world!

Most russians are not communists. While I like the jokes about this, in serious conversation I also have to notice that we suffered a lot because of communist regime.

**Reply by amigdala999** [reply]
And now we are suffering a lot bcs of next one regime. And we'll suffering again bcs of one after him. And the next one too. Where are always people who suffering of something.

**Comment by AKonia** [comment]
@kotovalexarian Когда интересно госкапитализм вдруг стал коммунизмом то ? Были элементы социализма, как общества высоких социальных гарантий - бесплатной качественной медицины, образования и науки на благо большинства, роста реальныз доходов но и то это всё начали сворачивать с Хрущёва - начали капиталистические преобразования и раздемокартизацию общества замылив глаза откровенной халявой-подачками и космосом. В целом же люди требующие от государства пенсий, медицины, безопасности и пр. - это уже социалисты, хотя и не коммунисты.

**Comment by TheShich** [comment]
Because RMS is our national hero, we believe in St. IGNUcius and FAIF is The Book!

**Comment by makpif** [comment]
Russian do not like LGBT.

**Reply by Bluesboy** [reply]
Not true at all. We indifferent about them.

**Reply by amigdala999** [reply]
Russians don't care much about LGBT. But Russia is a country made of countries populated with they own nations. With different religion, traditions etc. And these nations might be not so tolerant to LGBT as most russians are. You can be a gay in Moscow but may commit a suicide by identify yourself as a gay in other regions. So, government will need a real military operation to enforce LGBT related laws across 1/6 of a planet. And it's just 15 years since the last civil war ended in our territory. While it looks like more wars are comming right now... And even most of the russians don't care about LGBT they are against it bcs these changes will require too much efforts and risks without any profit for majority. What wewill get if we let 2 guys to adopt a children? Money? Economy growth? US will cancel its sanctions? We'll spill a blood for nothing. We're not in the position to make such generous gestures nowadays. We're underdogs. We need to survive. By unity. And LGBT isn't a thing that unite sociaty here. Any politics will explain you that.

**Reply by AKonia** [reply]
We don't approve LGBT it in form of intrusive propaganda, and totally disapprove it, when propaganda aimed to children, but in general, nobody tries to meddle in the personal life of other people. So we don't like LGBT only when LGBT activists becomes aggressive or pedofilic.

**Comment by TheShich** [comment]
As everywhere, some do love, some just say that they love, some just say that they don't love, some don't love and some of them are.

**Comment by zgwerby** [comment]
> Russian do not like LGBT.

Nope. Russians don't like "pidors".
Is very critical difference against def "gays" (or other non-traditional people) and def "pidors" (this is an extreme rough word "fag/faggot" version).  LGBT-people can be pidor, but not all gays - pidors, and not all pidors - gays. 
Now pidors attacked RMS, normal gays are out of politics. They do not have offend. But that stinky minority dirty everyone in сommutity. 
And on offended water is carried. 

All hail Mordor! Long live the RMS!

**Reply by KOLANICH** [reply]
>normal gays are out of politics

I don't think so. Normal gays are not out of politics, they are just not hysterical attention whores who try to get the level of attention above the one other persons get not by really deserving it, but by strange behaviour usually making an accent on their different orientation, like rallying naked with rainbow flags or like making hysterics on every occasion.

**Reply by zgwerby** [reply]
Nope again...Still, it is hard to explain to the English-speaking man the difference between the many "LGBTABCDEFGHIJKLMNOP member" and simply "pidor". And that all these Bassett's is a pidors, and tg Rowe is just a non-traditional man. 

> they are just not hysterical attention whores who try to get the level of attention above the one other persons 
Anyone can be adjusted under hysterical attention - even a person, just requiring attention to the problem of they city, they city house  or permanent flooding challenges from above: all formal criteria are - the annoying, to protrude your problem, the sacred rage to people with other problems, willing to everyone Problem 

Чукча не писатель, чукча читатель.
All hail Mordor! Long live the RMS!

**Comment by franpoli** [comment]
Let's try to catch up on Russian signatures and make this question obsolete

**Reply by Bluesboy** [reply]
Good luck with that, my friend. Sincerely.

**Reply by an-dev** [reply]
lmao

**Reply by AKonia** [reply]
So then you need to post info about this support-letter on big sites like reddit+twitter and some IT-aggregators that popular in your country, in case of Russia it was posted on really popular it-sites(habr.com, opennet.ru) and big IT-communities in social networks

**Reply by KOLANICH** [reply]
... and not get flagged by SJW ...

**Reply by AKonia** [reply]
@KOLANICH ... so possibly posting on twitter is a bad idea..
